app/src/main/java/com/infomaniak/drive/ui/fileList/SelectFolderActivity.kt

```diff
@@ -57,26 +57,34 @@ class SelectFolderActivity : BaseActivity() {
         val customArgs = navigationArgs.customArgs
         val currentFolderId = navigationArgs.folderId.getIntOrNull()
         val disabledFolderId = navigationArgs.disabledFolderId.getIntOrNull()
-        val currentUserDrive = UserDrive(userId, driveId)
+        DriveInfosController.getDrive(driveId = driveId, maintenance = false)?.let { currentDrive ->
```

Comment:
I think `selectedDrive`would suit better than `currentDrive`because it's the drive that is chosen by the user in the activity instead of the current drive on which he really is

Reply:
That would permit to remove the` this.` also